process/factory/metachain/vmContainerFactory.go

```diff
@@ -183,6 +199,34 @@ func (vmf *vmContainerFactory) createSystemVM() (vmcommon.VMExecutionHandler, er
 	return systemVM, nil
 }
 
+func (vmf *vmContainerFactory) createSystemVM() (vmcommon.VMExecutionHandler, error) {
+	scFactory, systemEI, err := vmf.createSystemVMFactoryAndEEI()
+	if err != nil {
+		return nil, err
+	}
+
+	vmf.systemContracts, err = scFactory.Create()
+	if err != nil {
+		return nil, err
+	}
+
+	return vmf.finalizeSystemVMCreation(systemEI)
+}
+
+func (vmf *vmContainerFactory) createSystemVMForGenesis() (vmcommon.VMExecutionHandler, error) {
```

Comment:
maybe add comment that this is mainnet genesis - july 31 - plus add a comment in config that delegation manager enable epoch should not be 0.

Reply:
done